### PR TITLE
Python wheel: failed to set RPATH for 3rd parties

### PR DESCRIPTION
**Pull request by slyubimt**
### Details:
 - *filter files for patchelf/install_name_tool:  skip symlinks and linker scripts*
 
### Tickets:
 - *CVS-51355*

### CODE REVIEW

File: inference-engine/ie_bridges/python/wheel/setup.py [hunk 1]
```diff
@@ -214,6 +229,10 @@ def set_rpath(rpath, executable):
     print(f'Setting rpath {rpath} for {executable}')  # noqa: T001
     cmd = []
     rpath_tool = ''
+    with open(os.path.realpath(executable), 'rb') as file:
+        if file.read(1) != b'\x7f':
+            log.warn(f'WARNING: {executable}: missed ELF header')
+            return
```
Comment (ilya-lavrenov) [inline inference-engine/ie_bridges/python/wheel/setup.py:235]: does it work for Darwin? They don't use elf format

Comment (ilya-lavrenov) [reply inference-engine/ie_bridges/python/wheel/setup.py:235]: Maybe we can use this fact?
```
$ cat libtbb.so
INPUT (libtbb.so.2)
```

Comment (ilya-lavrenov) [reply inference-engine/ie_bridges/python/wheel/setup.py:235]: E.g.:
```
(openvino) sandye51@sandye51-pc:~/Documents/Programming/git_repo/dldt-temp/tbb/lib$ ls -1 | xargs file
libhwloc.so.5:           ELF 64-bit LSB shared object, x86-64, version 1 (SYSV), dynamically linked, BuildID[sha1]=a3d8f4531c99d1c6ac96e209b3cfc2542c7e651a, stripped
libtbbbind.so:           ASCII text
libtbbbind.so.2:         ELF 64-bit LSB shared object, x86-64, version 1 (SYSV), dynamically linked, BuildID[sha1]=571945970aa0bfdc234c3a6ef9ff2b572f3cd511, stripped
libtbbmalloc_proxy.so:   ASCII text
libtbbmalloc_proxy.so.2: ELF 64-bit LSB shared object, x86-64, version 1 (SYSV), dynamically linked, BuildID[sha1]=dacf61b77712d8779b602f2f9d60e59b0648eb53, stripped
libtbbmalloc.so:         ASCII text
libtbbmalloc.so.2:       ELF 64-bit LSB shared object, x86-64, version 1 (SYSV), dynamically linked, BuildID[sha1]=2c509da2d728735ab1fd2181e6ad9e11b66981fc, stripped
libtbb_preview.so:       ASCII text
libtbb_preview.so.2:     ELF 64-bit LSB shared object, x86-64, version 1 (SYSV), dynamically linked, BuildID[sha1]=7451f3019f701fb3bfdff6089be40b22e4c49fc1, stripped
libtbb.so:               ASCII text
libtbb.so.2:             ELF 64-bit LSB shared object, x86-64, version 1 (SYSV), dynamically linked, BuildID[sha1]=15f747c1fa21d16e9afea9f1782ea806854bc990, stripped
```

Comment (slyubimt) [reply inference-engine/ie_bridges/python/wheel/setup.py]: You are right ELF check shall be moved under linux, since on macOS we have Mach-O.

**Review by ilya-lavrenov — APPROVED** [review]

**Comment by slyubimt** [comment]
@mryzhov it's a separate case. will be fixed in a scope of 51357 ticket

**Pull request merged**